src/main/resources/db/changelog/changes/12-create-orders-table.yaml

```diff
@@ -0,0 +1,81 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-orders-table
+      author: JS
+      changes:
+        - createTable:
+            tableName: orders
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+                    foreignKeyName: fk_orders_user
+                    references: users(id)
+              - column:
+                  name: status
+                  type: varchar(50)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: total
+                  type: decimal(19,2)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: order_date
+                  type: timestamp
+                  constraints:
+                    nullable: false
+              - column:
+                  name: shipping_address
+                  type: varchar(255)
+                  constraints:
+                    nullable: true
```

Comment:
Ensure shipping_address is consistent across the schema `nullable: true` and the DTO `@NotNull`.